src/NLog/Targets/ColoredConsoleTarget.cs

```diff
@@ -177,14 +185,31 @@ protected override void CloseTarget()
             base.CloseTarget();
         }
 
-            /// <summary>
+        /// <summary>
         /// Writes the specified log event to the console highlighting entries
         /// and words based on a set of defined rules.
         /// </summary>
         /// <param name="logEvent">Log event.</param>
         protected override void Write(LogEventInfo logEvent)
         {
-            this.Output(logEvent, this.Layout.Render(logEvent));
+            if (!FastModus)
```

Comment:
From a readability angle this condition could be easier to understand if inverted: if(FastModus){...}